services/parser-database/app.py

```diff
@@ -7,8 +7,8 @@
 from flask import jsonify  # pylint: disable=import-error
 
 from parser import parse_all_documents
-from connector import get_articles_that_match_keywords
-from connector import get_article_by_number
+from connector import get_article_by_id_db
+from connector import get_articles_that_match_keywords_db
```

Comment:
As @anniefu mentioned, you'll probably want to just have a `db_connector.py` that defines methods with the same names that you already use in the code, so that you can just change the import statement and not have to change any code.